app/helpers/application_helper.rb

```diff
@@ -71,6 +71,8 @@ def current_user_info_and_logout_link
   end
 
   def current_user_full_name
+    return 'Unknown' unless valid_user?(@current_user)
+
     [@current_user.given_name, @current_user.family_name].join(' ')
```

Comment:
Thanks for looking at this - looks like the underlying issue is whether the user was logged.

In investigating this, it did highlight that we make an assumption about the presence of both of these name values.

With that in mind can a safety net be added around these, ie

```ruby
[@current_user.given_name, @current_user.family_name].map(&:presence).join(' ') || ''
```

Or

```ruby
"#{@current_user.given_name} #{@current_user.family_name}"
```

Thanks

Reply:
Ignore the above - sounds like `[nil, nil].join(' ')` works just fine anyway